src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/domain/move/MovePersister.kt

```diff
@@ -80,6 +80,8 @@ class MovePersister(
 
           val moveToSave = newMove.copy(
             moveType = moveWithJourneysAndEvents.moveType(),
+            events = moveEvents,
```

Comment:
This was the crux of the problem.  The moveToSave was/is being used to determine the state of a cancelled move which in turn creates a journey so it can be priced and the suppliers paid.

The thing to be aware this does not impact what gets persisted on the actual move as these attributes are transient on the Move entity.